app/models/user.rb

```diff
@@ -500,6 +500,11 @@ def reload(...)
     super(...)
   end
 
+  # User was flagged by TMX as fraudulent, AND failed at the Post Office.
+  def double_failed?
+      fraud_review_pending? && %w[cancelled failed].include?(in_person_enrollments&.last&.status)
```

Comment:
interesting syntax i haven't used before. Am I correct in understanding it's checking if the status is "cancelled" or "failed"?

Reply:
Exactly. It's basically `...status == 'cancelled' || ...status == 'failed'`.

It always feels slightly backwards to me: "is the status in [array]?" feels way more natural than "Does [array] include the status?" But it's easy for Array/Enumerable to implement `include?`, versus having virtually every class implement `in?`.